package.json

```diff
@@ -7,7 +7,8 @@
     "start": "node app",
     "env": "cp env.dist .env",
     "db": "knex migrate:latest && knex seed:run",
-    "test": "lab"
+    "test": "npm run test:setup && lab -t 100 -c --verbose --colors --assert code --timeout 5000",
+    "test:setup": "npm run test:droptables && npm run test:createtables && npm run test:insertdata",
```

Comment:
This is incomplete. I was beginning to add this stuff, so don't expect it to work :sob: